scripts/vyatta-save-config.pl

```diff
@@ -133,7 +133,7 @@
             $host = $2;
         }
 
-        $rc = system("curl -u $user -# -T $url_tmp_file $save_file");
+        $rc = system("curl --insecure -u $user -# -T $url_tmp_file $save_file");
```

Comment:
I completely agree with Christian here. We should make it an option that one can pass to the command explicitly. 

Insecure by default _could_ be fine for image upgrade because there's also signature verification. But for saving configs to remote servers, rogue servers and address spoofing is a real concern, and with `--insecure` by default the router will silently play along with any rogue server.

Reply:
@c-po @dmbaturin 
I didn't find a correct way to add `node insecure` as it doesn't use regular templates.